dual_net.py

```diff
@@ -41,14 +41,17 @@
 # How many positions can fit on a graphics card. 256 for 9s, 16 or 32 for 19s.
 TRAIN_BATCH_SIZE = 256
 
+# The shuffle buffer size determines the max 'distance' an example ends up from
```

Comment:
I found this comment a bit obtuse. What are we shuffling? What do you mean by distance?

Reply:
It's also definitely not max; just some vague notion of gaussian distribution with stdev

Reply:
the max it *could* end up, then ;P  But yes, i'm either a) explaining the entire semantics of tf.Dataset.shuffle here, or b) leaving a comment like "shuffle buffer size: the size of the shuffle buffer".

Reply:
comment updated.